activities_sticky_queues/starter.py

```diff
@@ -0,0 +1,27 @@
+import asyncio
+from uuid import uuid4
+
+from temporalio.client import Client
+
+from activities_sticky_queues.tasks import FileProcessing
+
+
+async def main():
+    # Connect client
+    client = await Client.connect("localhost:7233")
+
+    # Start 10 concurrent workflows
+    futures = []
+    for _ in range(10):
+        handle = client.execute_workflow(
```

Comment:
```suggestion
        coro = client.execute_workflow(
```

Or "result" or something. Just to show it's not a `handle` like `start_workflow` would return.